packages/labs/testing/src/lib/fixtures/ssr-fixture.ts

```diff
@@ -64,7 +66,11 @@ export async function ssrFixture<T extends HTMLElement>(
       // ssrFixture@http://localhost:8000/lib/fixtures/ssr-fixture.js:19:34
       // ssrNonHydratedFixture@http://localhost:8000/lib/fixtures/ssr-fixture.js:98:45
       // @http://localhost:8000/test/my-element_test.js:20:37
-      [...(stack?.matchAll(/http:\/\/localhost:?[^:)]+/gm) ?? [])]
+      [
+        ...(stack?.matchAll(
+          /http:\/\/(localhost|host.containers.internal):?[^:)]+/gm
```

Comment:
same question as above
```suggestion
          /http:\/\/(localhost|host\.containers\.internal):?[^:)]+/gm
```